Installing-Oppia-(Mac-OS;-Python-3).md

```diff
@@ -150,69 +169,56 @@ For your vitual environment, we recommend you use [pyenv](https://github.com/pye
 
 2. Reload your shell or open a new terminal window to load your updated `~/.bash_profile` or `~/.zshrc`.
 
-3. Install Python 3.8.12 and the associated pip using the command below: 
-
-    `pyenv install 3.8.12`.
- 
-    This is how it looks like:  
+3. Install Python 3.8.12:
 
    ```console
    $ pyenv install 3.8.12
-   Installing openssl-1.1.1k.. 
+   Installing openssl-1.1.1k..
    Installed openssl-1.1.1k to /Users/user/.pyenv/versions/3.8.12
 
-   Installing readline-8.1... 
+   Installing readline-8.1...
    Installed readline-8.1 to /Users/user/.pyenv/versions/3.8.12
 
-   Downloading Python-3.8.12.tar.xz... 
-   -> https://www.python.org/ftp/python/3.8.12/Python-3.8.12.tar.xz 
-   Installing Python-3.8.12... 
-   patching file Misc/NEWS.d/next/Build/2021-10-11-16-27–38.bpo-45405.iSfdw5.rst 
-   patching file configure 
-   patching file configure.ac 
-   python-build: use zlib from xcode sdk 
+   Downloading Python-3.8.12.tar.xz...
+   -> https://www.python.org/ftp/python/3.8.12/Python-3.8.12.tar.xz
+   Installing Python-3.8.12...
+   patching file Misc/NEWS.d/next/Build/2021-10-11-16-27–38.bpo-45405.iSfdw5.rst
+   patching file configure
+   patching file configure.ac
+   python-build: use zlib from xcode sdk
    Installed Python-3.8.12 to /Users/user/.pyenv/versions/3.8.12
    ```
 
-4. Create a virtual environment for oppia using the command below: 
-
-    `pyenv virtualenv 3.8.12 oppia`
+4. Install Python 2.7.18:
 
-     This is how it looks like: 
 
    ```console
-   $ pyenv virtualenv 3.8.12 Oppia
-   ...
-   $ pyenv versions
-   ...
-   Oppia
+   $ pyenv install 2.7.18
    ...
+   Installed Python-2.7.18 to ~/.pyenv/versions/2.7.18
    ```
 
-   In the cloned `oppia` folder, run
+   This is needed because App Engine's `dev_appserver.py` [requires Python 2.7 to be installed](https://cloud.google.com/appengine/docs/standard/python3/tools/local-devserver-command).
+
+5. Make both Python 3 and Python 2 available globally on your system:
 
    ```console
-   pyenv local oppia
+   pyenv global 3.8.12 2.7.18
    ```
 
-   Now whenever you are within the `oppia` folder, the virtual environment will be active.
+   Note that you can use a different version of Python 3 if you prefer--we'll specify Python 3.8.12 as our version for Oppia development later.
 
-5. Install the Python dependencies:
+5. Create a virtual environment for oppia by making a file `opensource/.envrc` with the following contents:
```

Comment:
Users may assume that `opensource/.envrc` is the file name. Can you specify the folder and file name separately?

Reply:
We assume throughout our installation instructions that users understand path notation. Do you think we need to re-evaluate that assumption?

Reply:
Yes, I have observed a few devs assuming that to be the filename itself and not the path/filename.